common/lib/capa/capa/inputtypes.py

```diff
@@ -310,6 +311,8 @@ def _get_render_context(self):
             (a, v) for (a, v) in self.loaded_attributes.iteritems() if a in self.to_render
         )
         context.update(self._extra_context())
+        if self.answervariable:
+            context.update({'answervariable': self.answervariable})
```

Comment:
This would be a bit cleaner: `context['answervariable'] = self.answer_variable`.